lib/inputstreamhelper/__init__.py

```diff
@@ -413,7 +420,7 @@ def _has_inputstream(self):
     def _inputstream_enabled(self):
         """Returns whether selected InputStream add-on is enabled.."""
         data = execute_jsonrpc(dict(jsonrpc='2.0', id=1, method='Addons.GetAddonDetails', params=dict(addonid=self.inputstream_addon, properties=['enabled'])))
-        if data['result']['addon']['enabled']:
+        if data.get('result') and data.get('result').get('addon').get('enabled'):
```

Comment:
You could also do:
```python
        if data.get('result', {}).get('addon', {}).get('enabled')
```
It is safer in case the addon-key would be missing as well.

Reply:
Okay, changed.